src/main/scala/codacy/checkstyle/Checkstyle.scala

```diff
@@ -42,6 +44,11 @@ object Checkstyle extends Tool {
     run(filesToLint, config)
   }
 
+  private def setCheckstyleConfigLocationProperty(configFilePath: String) = {
```

Comment:
Codacy found an issue: [Use active names for operations with side effects](https://app.codacy.com/gh/codacy/codacy-checkstyle/pullRequest?prid=5929160)